code/datums/mind.dm

```diff
@@ -122,15 +122,19 @@
 	if (hasFactionsWithHUDIcons())
 		update_faction_icons()
 
-/datum/mind/proc/store_memory(new_text)
-	if(length(memory) > MAX_PAPER_MESSAGE_LEN)
-		to_chat(current, "<span class = 'warning'>Your memory, however hazy, is full.</span>")
-		return
-	if(length(new_text) > MAX_MESSAGE_LEN)
-		to_chat(current, "<span class = 'warning'>That's a lot to memorize at once.</span>")
-		return
-	if(new_text)
-		memory += "[new_text]<BR>"
+/datum/mind/proc/store_memory(new_text, var/forced)
```

Comment:
don't structure it like this, instead just add forced to the other checks

Reply:
You can just put the other checks in an `if(!forced)`, since they return and end execution if they fail